lib/archethic/contracts/interpreter/common_interpreter.ex

```diff
@@ -353,6 +360,80 @@ defmodule Archethic.Contracts.Interpreter.CommonInterpreter do
     {new_node, acc}
   end
 
+  # BigInt mathematics to avoid floating point issues
+  # the `0.0 + x` is used to cast integers to floats
+  def postwalk(_node = {:*, meta, [lhs, rhs]}, acc) do
```

Comment:
I believe in elixir there is the operator `**` which is the power of a number that replace the erlang `:math.pow(x, y)` maybe you could also handle this operator

Reply:
There's no `pow` in the Decimal library yet. This may be done in a later issue.